packages/host/tests/integration/components/operator-mode-test.gts

```diff
@@ -1061,67 +1061,6 @@ module('Integration | operator-mode', function (hooks) {
     assert.dom('[data-test-field="friends"]').containsText('Mango');
   });
 
-  test('can create a new card to add to a linksToMany field with no existing value', async function (assert) {
```

Comment:
I think if we're planning on adding the functionality to `create new` after clicking `add friends` button, we should skip these tests instead of removing them, and create a ticket about it so we don't forget.